pkgs/applications/misc/web-media-controller/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, glib, pcre, json-glib }:
+
+stdenv.mkDerivation rec {
+  pname = "wmc-mpris";
+  name = "${pname}-unstable";
```

Comment:
`name` won't have a `version` if we do this.
Can you make `unstable` a part of `version. I don't think we've updated the docs for this.

Reply:
I'm not sure I understand your intention. Would you be satisfied with a date given as a pseudo version? Perhaps more similarly to the way this is done in [`xsel`](https://github.com/NixOS/nixpkgs/blob/962c1884d5b81434f80094e536c7b57f7b17330e/pkgs/tools/misc/xsel/default.nix#L4-L6)?

Reply:
```
pname = "wmc-mpris";
version = "unstable-2019-04-12";
```

Reply:
Fixed. Not exactly like in `xsel` but pretty close to it.

Reply:
You commited this as `2019-04-12-unstable` instead of  `unstable-2019-04-12`. By doing that it would be part of the version.

Probably should fix the date also https://github.com/f1u77y/wmc-mpris/commit/3b92847c576662732984ad791d6c7899a39f7787

Reply:
I think it's OK now.